Makefile

```diff
@@ -439,3 +441,47 @@ install-kuttl:
 mocks:
 	mockery --dir=./pkg/cassandra --output=./pkg/mocks --name=ManagementApiFacade
 	mockery --dir=./pkg/reaper --output=./pkg/mocks --name=Manager  --filename=reaper_manager.go --structname=ReaperManager
+
+
+# The image base name used in GKE clusters.
+GKE_IMAGE_TAG_BASE ?= us-docker.pkg.dev/community-ecosystem/$(IMAGE_TAG_BASE)
```

Comment:
Is it necessary to push images into a GKE Artifact repo? @bradfordcp where are we at with the quay.io migration? Once that is complete, could we just pull from there?